app/Jobs/RefreshNftMetadata.php

```diff
@@ -39,7 +38,7 @@ public function __construct(
      */
     public function handle(AlchemyWeb3DataProvider $provider): void
     {
```

Comment:
Something we can consider is to also dispatch a `FetchCollectionActivity` job here, so it's possible to manually refresh specific collections without having to wait for the backend.